### ISSUE TITLE
MatSnackBar does not announce its content to screen readers

### ISSUE THREAD

**Issue by joeysilva** [angular/components#13015]
I hear no announcement from the example snackbar using Mac VoiceOver: https://material.angular.io/components/snack-bar/examples

Even if I increase the duration, set an explicit announcementMessage, and/or set politeness to assertive, I still hear no announcement.

**Comment by joeysilva** [comment]
Mac VoiceOver (High Sierra 10.13.6) in Chrome.

Just tested in Safari, and it works there. Might be a Chrome issue I suppose.

**Comment by mfairchild365** [comment]
Can confirm that this is an issue. The aria-live region needs to exist in the dom before content is added to it. If the aria-live region and content are added to the dom at the same time, the aria-live region will not announce anything.

**Comment by jelbourn** [comment]
I believe this has been fixed in later versions, with #21137 tracking the remaining gap.
